nicoRDL/dotTheory-JS
# JS-102 (JS 346) JavaScript Numbers: The Math Object

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-346

## Description
-  completed Math Object tutorials
- finished all code challenges and quizzes

## Steps to Validate
1. Open the index.html file
2. A message prompt should immediately appear; input any number and press enter
3. Open the dev tools console; a random number should have been generated based on the input

Risks
N/A
